src/type_checker/checker.rs

```diff
@@ -24,6 +24,8 @@ where
 {
     pub kind: FnKind<B>,
     pub span: Span,
+    // todo: generics for inferred values
+    // - will be useful for builtins that will need to know the generic values
```

Comment:
I think TODOs should resist the test of time, I'm not sure if we'll understand what the TODO is here in like 6 months :D either add more info or link to an issue that adds more info

Reply:
this is removed in another PR